packages/mrgn-ui/src/components/action-box-v2/store/stake-box-store.ts

```diff
@@ -0,0 +1,31 @@
+import { create, StateCreator } from "zustand";
+import { persist } from "zustand/middleware";
+
+import { LstData } from "@mrgnlabs/mrgn-utils";
+
+interface StakeBoxState {
+  // State
+  lstData: LstData | null;
+
+  // Actions
+  setLstData: (lstData: LstData) => void;
+}
+
+function createStakeBoxStore() {
```

Comment:
let's remove this store, and just store lstData in the actionbox store

seems a bit redundant to create a seperate store for this

Reply:
Dont you think its cleaner to have this in a separate store, since this is only used in the stakebox? 

Or should we conditionally fetch on the actiontype? Else it'll be done for every actionbox instance

Reply:
sorry let me back track, just looked into the code a bit more in depth

We're using the context provider to fetch and store the lstData, but the provider also stores the the lstData. So we're storing in two places.

In action-box.tsx the useStakeBoxContext hook is used to insert the lstData into the props of the StakeBox component but in the StakeBox it is not being used and just fetched from the store.

So I think the best move is to get rid of the stake-box store and just store in inside the context with a simple useState. Otherwise it would be kind of pointless to create a context provider without using the context, and you don't have the overhead of a zustand store for a single value,

Reply:
Got it! Will update to remove the store & just use the context directly